esp/bindings/http/platform/httpbinding.hpp

```diff
@@ -320,7 +333,36 @@ class esp_http_decl EspHttpBinding :
         }
         return false;
     }
-    ISecManager* querySecManager() {return m_secmgr.get(); }
+    ISecManager* querySecManager() { return m_secmgr.get(); }
+    IAuthMap* queryAuthMAP() { return m_authmap.get();}
+    const char* queryAuthMethod() { return m_authmethod.str(); }
```

Comment:
My previous comment (above), was suggesting you made the above method and others like it const methods, not to make the return scalar a const. There's no point in the const on a scalar return type (some compilers give warning apparently).

So e.g. 
```
    const AuthType getDomainAuthType() { return domainAuthType; }
```
to:
```
    AuthType getDomainAuthType() const { return domainAuthType; }
```